docs/run-prebuilt-wda.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Run Prebuilt WebDriverAgentRunner
+---
+
+
+https://github.com/appium/WebDriverAgent/releases
+
+
+## `appium:useXctestrunFile` and `appium:bootstrapPath`
```

Comment:
here as well:
Use the following capabilities in order to tell your session to use the prebuilt WDA binaries 

...

Set bootstrapPath to the full path of the folder where the corresponding prebuilt binary is located on the server file system
...